### DISCUSSION TITLE
consider adding bots to the tutorial

### DISCUSSION THREAD

**Discussion by alee** [virtualcommons/port-of-mars#823] [Ideas]
would highlight full gameplay for the users

**Comment by chrstngyn** [comment]
https://www.filamentgames.com/blog/10-tips-designing-game-tutorial/

https://en.wikipedia.org/wiki/The_Magical_Number_Seven,_Plus_or_Minus_Two

https://www.filamentgames.com/blog/5-assessment-strategies-learning-games/

https://mikelaraman.com/the-ultimate-guide-to-videogame-tutorial-design
